pkg/agent/route/route_linux.go

```diff
@@ -670,14 +799,148 @@ func (c *Client) syncIPTables() error {
 	return nil
 }
 
+func (c *Client) generateProxyAllRules(nodePortIPSet string,
+	loadBalancerIPSet string,
+	externalIPIPSet string,
+	serviceCIDR string,
+	nodePortDNATVirtualIP string,
+	serviceVirtualIP string,
+	isIPv6 bool) map[string]map[string][]string {
+	rules := map[string]map[string][]string{
+		iptables.NATTable: make(map[string][]string),
+		iptables.RawTable: make(map[string][]string),
+	}
+	var natPreroutingRules, natOutputRules []string
+	if c.kubeServiceHost != "" {
```

Comment:
I suppose you can skip this check? since utilnet.IsIPv6String(c.kubeServiceHost) will return false when the string is empty?

Reply:
Make sense